libraries/chain/include/eosio/chain/abi_serializer.hpp

```diff
@@ -184,14 +195,8 @@ namespace impl {
    using path_item = static_variant<empty_path_item, array_index_path_item, field_path_item, variant_path_item>;
 
    struct abi_traverse_context_with_path : public abi_traverse_context {
-      abi_traverse_context_with_path( const abi_serializer& abis, fc::microseconds max_serialization_time, const std::string_view& type )
-      : abi_traverse_context( max_serialization_time ), abis(abis)
-      {
-         set_path_root(type);
-      }
-
-      abi_traverse_context_with_path( const abi_serializer& abis, fc::microseconds max_serialization_time, fc::time_point deadline, const std::string_view& type )
-      : abi_traverse_context( max_serialization_time, deadline ), abis(abis)
+      abi_traverse_context_with_path( const abi_serializer& abis, abi_serializer::yield_function_t yield, const std::string_view& type )
```

Comment:
const abi_serializer::yield_function_t& yield

Reply:
It is moved.